sass/utilities/mixins.sass

```diff
@@ -173,8 +173,10 @@
 // Responsiveness
 
 $tablet: 769px !default
-$desktop: 1000px !default // 960px container + 40px
-$widescreen: 1192px !default // 1152px container + 40
+// 960px container + 40px
+$desktop: 1000px !default
+// 1152px container + 40
+$widescreen: 1192px !default
```

Comment:
This is the error that I have with the comments in the same line that the variable definition

![image](https://cloud.githubusercontent.com/assets/3193604/22035595/44c88830-dccf-11e6-9c1e-78b4c46c3996.png)